packages/web/src/pages/sign-up-page/pages/PickHandlePage.tsx

```diff
@@ -162,46 +162,44 @@ export const PickHandlePage = () => {
       onSubmit={handleSubmit}
       validateOnChange={false}
     >
-      <Page as={Form} centered={!isMobile} transitionBack='vertical'>
-        {isWaitingForSocialLogin ? (
-          <SocialMediaLoading />
-        ) : (
-          <>
-            <Heading
-              prefix={
-                isMobile ? null : <OutOfText numerator={1} denominator={2} />
-              }
-              heading={messages.title}
-              description={messages.description}
-              centered={!isMobile}
+      {isWaitingForSocialLogin ? (
+        <SocialMediaLoading />
+      ) : (
+        <Page as={Form} centered={!isMobile} transitionBack='vertical'>
```

Comment:
Nothing wrong with it but for my own curiosity, what the reason is for moving it out of the page?

Reply:
so for some reason it messed with the container styling. this was thi simple change so i went for it rather than figure out another solution :)